extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/providers/KnownOidcProviders.java

```diff
@@ -98,4 +100,24 @@ private static OidcTenantConfig apple() {
         ret.getCredentials().getJwt().setAudience("https://appleid.apple.com/");
         return ret;
     }
+
+    private static OidcTenantConfig spotify() {
+        // See https://developer.spotify.com/documentation/general/guides/authorization/code-flow/
+        OidcTenantConfig ret = new OidcTenantConfig();
+        ret.setDiscoveryEnabled(false);
+        ret.setAuthServerUrl("https://accounts.spotify.com/");
```

Comment:
Judging from the twitter and github configs, you can remove the slashes there.
```suggestion
        ret.setAuthServerUrl("https://accounts.spotify.com");
```

Reply:
I copied from the apple config :)